lib/strings.nix

```diff
@@ -429,7 +429,7 @@ rec {
   */
   fixedWidthNumber = width: n: fixedWidthString width "0" (toString n);
 
-  /* Check whether a value is a store path.
+  /* Check whether a string or a path is a store path.
```

Comment:
It’s still a value, since we’re returning a bool for every possible value. I’d change the docs to:

```
/* Check whether a value is a store path.
   Every literal path (e.g. `./foobar`) is a store path,
   every string that points to a file directly under `buildins.storeDir` as well.
*/
```